lib/cart.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "menu_item"
+require_relative "menu"
+require_relative "order_item"
+require_relative "customer"
+
+class Cart
+  attr_accessor :cart
+  attr_accessor :total_cost
```

Comment:
Yes, I noticed different people prefer it different ways.  I picked this way because of what I read on [this unofficial Ruby style guide](http://naildrivin5.com/ruby-style/), which made sense to me: 

> When declaring attributes with attr_reader or attr_accessor, put one per line
> Why? This makes it easier to add new attributes without creating complex diffs. It also documents what the attributes represent